numba/core/transforms.py

```diff
@@ -335,6 +336,8 @@ def dispatcher_factory(func_ir, objectmode=False, **kwargs):
             myflags.force_pyobject = True
             myflags.no_cpython_wrapper = False
             cls = ObjModeLiftedWith
+        elif dppl_mode:
+            cls = DPPLLiftedWith
```

Comment:
What is it actually doing?

Reply:
This is necessary to create a dispatcher specifically for lifted code with new semantics. This is for now an intermediate solution, in the future it will be necessary to rewrite it so as not to change the numba files.